pages/k8s/upgrade-notes.md

```diff
@@ -13,7 +13,40 @@ layout: [base, ubuntu-com]
 toc: False
 ---
 
-This page is intended to deal with specific, special circumstances you may encounter when upgrading between versions of the **Charmed Distribution of Kubernetes<sup>&reg;</sup>.** The notes are organised according to the upgrade path below, but also be aware that any upgrade that spans more than one minor version may need to beware of notes in any of the intervening steps.
+This page is intended to deal with specific, special circumstances you may
+encounter when upgrading between versions of the
+**Charmed Distribution of Kubernetes<sup>&reg;</sup>.** The notes are
+organised according to the upgrade path below, but also be aware that any
+upgrade that spans more than one minor version may need to beware of notes in
+any of the intervening steps.
+
+<a  id="1.14"> </a>
+
+## Upgrading to 1.14
+
+This upgrade includes support for **CoreDNS**. All new deployments of **CDK**
+with Kubernetes 1.14 will install **CoreDNS** by default instead of **KubeDNS**.
+
+Existing deployments that are upgraded to **CDK** and Kubernetes 1.14 will
```

Comment:
s/CDK and Kubernetes 1.14/CDK 1.14/

Reply:
Fixed, thanks

Reply:
you missed one, but I got it, and as a bonus I found a 'that' which should have been a 'which'

Reply:
:cake:

Reply:
Oops. Good catches, thanks!